### ISSUE TITLE
No Solution Stack named '64bit Amazon Linux 2016.03 v2.1.1 running Tomcat 8 Java 8' found.

### ISSUE THREAD

**Issue by tingletech** [hybox/aws#78]
The error above is in the fcrepo stack

humm, maybe I will need to fork the templates -- looks like the current solution stack is v2.1.3

http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/concepts.platforms.html#concepts.platforms.java

**Comment by cbeer** [comment]
I'm surprised they don't keep the old versions available. Definitely send a pull request with the necessary change. Thanks!

**Comment by tingletech** [comment]
This is probably not bundled right for how y'all do things, but the change did end it in this PR I did the other day with the readme tweak https://github.com/hybox/aws/pull/76

I've just done this:

```
aws s3 mb s3://uc-hybox-deployment-artifacts
aws s3 sync s3://hybox-deployment-artifacts s3://uc-hybox-deployment-artifacts
```

and upload my patched `fcrepo.json` into uc-hybox-deployment-artifacts.  I've just launched the stack off of uc-hybox-deployment-artifacts and will let you know how it goes

edit: I wonder if there is way to request the newest stack?

**Comment by tingletech** [comment]
also need to update `No Solution Stack named '64bit Amazon Linux 2016.03 v2.1.2 running Ruby 2.3 (Puma)' found.`; also goes to v2.1.3 http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/concepts.platforms.html#concepts.platforms.ruby

**Comment by cbeer** [comment]
Fixed in 728cadbdc18c9146f5d7358d4ce92090aba91b29
